### ISSUE TITLE
Lose ability to click on fractal window

### ISSUE THREAD

**Issue by degriz** [buddhi1980/mandelbulber2#859]
After what seems to be a random amount of time you can no longer click in the window to move camera or change any other settings. Latest nightly.

**Comment by buddhi1980** [comment]
Can you paste here this settings file which causes the problem?

**Comment by michas0602** [comment]
I have also recognized this in 1 file here! Latest dev.

When I am not completely wrong I have a file here where it happens all the time...possibly it helps to reproduce. But it takes 1-2 hours until I can take a look.

Furthermore there seems to be a very sporadic problem (with antialiasing ?) - sometimes I have strange results with latest dev, not sure how to reproduce, it seems it often happens in high resolutions with aa on. Hard to describe how they look, somehow unfinished - dark and unsharp. I'll try to reproduce later today, too.

In early 2.26 dev some months ago I was not able to see both problems, so I think they were caused by something in the last weeks somehow
